api/nodecontrol/client.go

```diff
@@ -175,7 +159,7 @@ func (c *ControlAPIClient) DeployWorkload(namespace, nodeId string, req nodegen.
 		return nil, err
 	}
 
-	msg, err := c.nc.Request(models.DeployRequestSubject(namespace, nodeId), req_b, DefaultRequestTimeout)
+	msg, err := c.nc.Request(models.DeploySubject(nodeId), req_b, DefaultRequestTimeout)
```

Comment:
should we call this a "direct deploy" to contrast against the "auction deploy"

Reply:
could even change the subject from `DEPLOY` to `DDEPLOY` to contrast `ADEPLOY`

Reply:
I can do that...